### ISSUE TITLE
Question: The best model is always the last one.

### ISSUE THREAD

**Issue by jonghajang** [keras-team/autokeras#1366]
Hello

When I load the best model, the last epoch model of the last trial is always loaded. What is the problem? 
Here is the code:

clf = ImageClassifier(seed=123,tuner='bayesian',loss='categorical_crossentropy',metrics= 'accuracy',
    overwrite=True,objective='val_accuracy',project_name='autokeras2',
    max_trials=3)
# Feed the image classifier with training data.
clf.fit(train_images, y_train, validation_data=(val_images,y_val),epochs=5,batch_size=100)
mm = clf.tuner.get_best_model()

![asdfasdf](https://user-images.githubusercontent.com/66405055/95163122-2b932900-07e2-11eb-83e8-2d6280d106a6.PNG)

In the capture, the performance of the best accuracy model is 0.55, but the best model is always the last epoch model of the last trial (for example, the epoch is 5).

![image](https://user-images.githubusercontent.com/66405055/95163310-8cbafc80-07e2-11eb-8b1c-a841f5d98c13.png)

Thank you.

**Comment by jonghajang** [comment]
The working environment is
autokeras version : 1.0.9
tensorflow version : 2.0.3

thank you.

**Comment by ciessielski** [comment]
What is the prediction you get? Maybe it's a list with one value which is the most common...

**Comment by haifeng-jin** [comment]
It was not the last trial, it is actually the final fit.
We will always retrain the model if you specifies the epochs.

If you want to get that model, you can use `clf.tuner.get_best_models(3)`.
